sklearn/metrics/tests/test_ranking.py

```diff
@@ -371,6 +371,33 @@ def test_roc_curve_drop_intermediate():
                               [1.0, 0.9, 0.7, 0.6, 0.])
 
 
+def test_roc_auc_score_pos_label():
+    # ensure the correctness of pos_label parameter in roc_auc_score
+    y_true_1 = [0, 1, 1, 0]
+    y_true_2 = [1, 2, 2, 1]
+    y_true_3 = ['False', 'True', 'True', 'False']
+    y_pred = [0.1, 0.9, 0.2, 0.8]
+    roc_auc_score_1 = roc_auc_score(y_true_1, y_pred)
+
+    # int pos_label and binary y_true
+    roc_auc_score_2 = roc_auc_score(y_true_1, y_pred, pos_label=1)
+    assert_almost_equal(roc_auc_score_1, roc_auc_score_2)
+    roc_auc_score_3 = roc_auc_score(y_true_1, y_pred, pos_label=0)
+    assert_almost_equal(roc_auc_score_1, 1-roc_auc_score_3)
```

Comment:
spaces around `-`, please.

PEP8 only recently allowed not having spaces around binary operators, but spaces should only be removed when the grouping is visually helpful, in contrast to other operators with spaces (e.g. `a*b + c`). It is not, here.